l10n_br_cnpj_search/models/res_partner_wizard.py

```diff
@@ -0,0 +1,90 @@
+from odoo import models, fields, api
+from erpbrasil.base.misc import punctuation_rm
+from requests import get
+
+
+class ResPartnerWizard(models.TransientModel):
+    _name = "open.search.wizard"
+
+    cnpj_cpf = fields.Char(string="CNPJ")
+    legal_name = fields.Char(string="Legal name")
+    name = fields.Char(string="Name")
+    inscr_est = fields.Char(string="IE")
+    zip = fields.Char(string="ZIP")
+    street_name = fields.Char(string="Street Name")
+    street_number = fields.Char(string="Street Number")
+    street2 = fields.Char(string="Complement")
+    district = fields.Char(string="District")
+    state_id = fields.Many2one('res.country.state', string="Stade")
+    city_id = fields.Many2one(
+        string="City",
+        comodel_name="res.city",
+        domain="[('state_id', '=', state_id)]",)
+    country_id = fields.Many2one('res.country', string="Country")
+    vat = fields.Char(string="Vat")
+    phone = fields.Char(string="Phone")
+    mobile = fields.Char(string="Mobile")
+    email = fields.Char(string="E-mail")
+    website = fields.Char(string="Website")
```

Comment:
- Falta o campo do proprio partner_id que será pego no contexto, conforme falei no outro comentário. Pode ser readonly e não esquece de definir na view.
- Quando o nome da string é exatamente igual ao nome do campo não é preciso ter uma string definida. (_ são substituidos por espaço e as primeiras letras de cada palavra se tornam maiúsculas automaticamente, ex: street_number se torna Street Number)
- Falta alguns campos que são trazidos pelas pesquisas: legal_nature, equity_capital, cnae_main_id, cnae_secondary_ids

Reply:
Eu não fiz a criação do campo partner_id por que não conseguir passar ali no Odoo não trouxe o default_partner_id, fiz as mudanças de tirar as strings definida e também a criação dos campos que faltava e adicionando eles na view
OBS: campo cnae_secondary_ids esta comentado por que ele ta retornando uma lista de ID então to vendo como posso tratar isso